modules/network-annotation-processor/src/main/java/org/apache/ignite/internal/network/processor/messages/MessageBuilderGenerator.java

```diff
@@ -89,19 +93,29 @@ private List<MethodSpec> generateGettersAndSetters(MessageClass message, List<Ex
         for (ExecutableElement field : fields) {
             String fieldName = field.getSimpleName().toString();
 
-            TypeMirror type = field.getReturnType();
+            TypeMirror returnTypeMirror = field.getReturnType();
+            TypeName returnTypeName = TypeName.get(returnTypeMirror);
+
+            if (!isMethodReturnPrimitive(field) && isMethodReturnNullableValue(field)) {
```

Comment:
> Please use methodReturns* instead of isMethodReturn*, I struggle to read current names.

Fix it.

> It's also weird to pass field as a method, and interpret field type as a return type, but that's how javapoet works I guess?

This name was not entered by me, I will try to fix it.